buffer/src/main/java/io/netty/buffer/ByteBuf.java

```diff
@@ -280,6 +295,7 @@
      * @deprecated use the Little Endian accessors, e.g. {@code getShortLE}, {@code getIntLE}
      * instead of creating a buffer with swapped {@code endianness}.
      */
+    @Deprecated
```

Comment:
Maybe not really important but this belongs not really to this PR and may be better in an extra PR  (also the above).